### PR TITLE
Show different labels for ParaTime snapshot charts

### PR DESCRIPTION
**Pull request by buberdds**
Based on Don's feedback we want to keep current charts behaviour, but we need to show different label for ALL_TIME filter.

### CODE REVIEW

File: src/app/pages/ParatimeDashboardPage/ActiveAccounts.tsx [hunk 1]
```diff
@@ -57,8 +58,16 @@ type ActiveAccountsProps = {
   chartDuration: ChartDuration
 }
 
+const getLabels = (t: TFunction): Record<ChartDuration, string> => ({
+  [ChartDuration.TODAY]: t('chartDuration.lastHour'),
+  [ChartDuration.WEEK]: t('chartDuration.lastDay'),
+  [ChartDuration.MONTH]: t('chartDuration.lastDay'),
+  [ChartDuration.ALL_TIME]: t('chartDuration.lastMonth'),
```
Comment (csillag) [inline src/app/pages/ParatimeDashboardPage/ActiveAccounts.tsx:65]: This mapping looks weird? (Smells like, off by one?)

File: src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx [hunk 2]
```diff
@@ -15,12 +16,20 @@ import { PercentageGain } from '../../components/PercentageGain'
 import startOfHour from 'date-fns/startOfHour'
 import { useRequiredScopeParam } from '../../hooks/useScopeParam'
 
+const getLabels = (t: TFunction): Record<ChartDuration, string> => ({
+  [ChartDuration.TODAY]: t('chartDuration.lastHour'),
+  [ChartDuration.WEEK]: t('chartDuration.lastDay'),
+  [ChartDuration.MONTH]: t('chartDuration.lastDay'),
+  [ChartDuration.ALL_TIME]: t('chartDuration.lastDay'),
+})
+
```
Comment (csillag) [inline src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx:25]: Why do we have this mapping repeated by card?

Comment (buberdds) [reply src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx:25]: We want to have different label for each chart.

**Comment by csillag** [comment]
So, as far as I can tell, two things happened here together:
 - There was a small refactoring, changing where the chart duration labels are calculated
 - There was (probably) some change in the calculation

But since the two changes are conflated in one commit, it's hard for me to figure out what was the actual behavior change....

File: src/app/pages/ParatimeDashboardPage/ActiveAccounts.tsx (hunk 1 above)
Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/ActiveAccounts.tsx:65]: OK I get it now after the explanation

File: src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx (hunk 2 above)
Comment (csillag) [reply src/app/pages/ParatimeDashboardPage/TransactionsChartCard.tsx:25]: Ahh I see, the very last item is different. (I would probably still prefer to have a common function with a parameter, but whatever.)

**Review by csillag — APPROVED** [review]
Reviewed. LGTM

**Pull request merged**